### PR TITLE
Add drivers for WPE based browsers

### PR DESCRIPTION
**Pull request by psaavedra**
* Add a driver for cog running on the top of the WPE backend for FDO
* Add  a driver for MiniBrowser running with WPE

### CODE REVIEW

File: webkitpy/benchmark_runner/browser_driver/linux_cogfdo_driver.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+from linux_browser_driver import LinuxBrowserDriver
+
+class CogFdoBrowserDriver(LinuxBrowserDriver):
+    browser_name = 'cog-fdo'
+    process_search_list = ['cog']
+
+    def launch_url(self, url, options, browser_build_path):
+        self._browser_arguments.append('-P fdo')
+        self._browser_arguments = [url]
```
Comment (aperezdc) [inline webkitpy/benchmark_runner/browser_driver/linux_cogfdo_driver.py]: This is replacing `self._browser_arguments`, isn't it? I believe that you need to `.append(url)` here, or use `.extend()` so you can put all in one line:

```python
    self._browser_arguments.extend(('--platform=fdo', url))
```

I also like it better passing the long version of the option (`--platform=fdo`), to make the code more self-documenting.

Comment (psaavedra) [reply webkitpy/benchmark_runner/browser_driver/linux_cogfdo_driver.py]: yes, I catched it right now running the requested changes by Carlos. Good catch anyway,

**Comment by psaavedra** [comment]
@aperezdc @clopez : JFYI I made  your requested changes

File: webkitpy/benchmark_runner/browser_driver/linux_cog_driver.py [hunk 2]
```diff
@@ -36,3 +36,11 @@ def launch_url(self, url, options, browser_build_path):
 
     def launch_driver(self, url, options, browser_build_path):
         raise ValueError("Browser {browser} is not available with webdriver".format(browser=self.browser_name))
+
+
+class CogFdoBrowserDriver(CogBrowserDriver):
+    browser_name = 'cog-fdo'
+
+    def launch_url(self, url, options, browser_build_path):
+        self._browser_arguments = ['-P fdo', url]
```
Comment (aperezdc) [inline webkitpy/benchmark_runner/browser_driver/linux_cog_driver.py]: The `-P fdo` command line arguments need to be *two* separate items. This needs to be:

```python
self._browser_arguments = ['-P', 'fdo', url]
```

or, event better, as it is more descriptive:

```python
self._browser_arguments = ['--platform=fdo', url]
```

**Comment by clopez** [comment]
LGTM, thanks @psaavedra !

**Pull request merged**